backend/src/routes/user.ts

```diff
@@ -180,7 +211,7 @@ router.get("/mentee/:userId", [verifyAuthToken], async (req: Request, res: Respo
           grade,
           topicsOfInterest,
           careerInterests,
-          whyPaired: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+          whyPaired,
```

Comment:
I can't actually add a comment to line 228 since it wasn't changed, but HTTP 405 is probably not the appropriate status to use here - 405 means [method not allowed](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405) (like the request used the HTTP DELETE method but the object can't be deleted).

This seems more like an internal error (500), since there aren't any users that aren't mentees or mentors (right?).

Reply:
done